build-tools/packages/build-tools/README.md

```diff
@@ -79,7 +79,7 @@ Options:
   -r --rebuild        Clean and build on matched packages (all if package regexp is not specified)
      --reinstall      Same as --uninstall --install.
      --root <path>    Root directory of the Fluid repo (default: env _FLUID_ROOT_)
-  -s --script <name>  npm script to execute (default:build)
+  -t --target <name>  target to execute (default:build)
```

Comment:
Do we have to change this?

Reply:
I can make both work the same given that it doesn't necessarily need to be "script"